backend/app/services/channels/handler.py

```
@@ -643,18 +649,42 @@ async def _handle_devices_command(
+
+            # Check if we have a valid Claude model for device mode
+            if not override_model_name:
+                model_selection = await model_selection_manager.get_selection(user.id)
+                if model_selection and not model_selection.is_claude_model():
+                    model_display = (
+                        model_selection.display_name or model_selection.model_name
+                    )
+                    await self.send_text_reply(
+                        message_context,
+                        f"⚠️ 当前模型 **{model_display}** 不支持设备模式\n\n"
+                        "设备模式仅支持 Claude 模型，请先使用 `/models` 切换到 Claude 模型",
```

Ruff (0.14.14)

[warning] String contains ambiguous `，` (FULLWIDTH COMMA). Did you mean `,` (COMMA)?

(RUF001)